packages/cozy-clisk/src/contentscript/ContentScript.js

```diff
@@ -165,8 +182,22 @@ export default class ContentScript {
     this.contentScriptType = contentScriptType
     log.info(`I am the ${contentScriptType}`)
 
+    if (!this.bridge) {
+      throw new Error(
+        'No bridge is defined, you should call ContentScript.init before using this method'
+      )
+    }
+
     if (contentScriptType === WORKER_TYPE) {
+      this.requestInterceptor?.on('response', response => {
+        this.bridge?.emit('workerEvent', {
+          event: 'requestResponse',
```

Comment:
Since these are events already coming from RequestInterceptor, I found it redundant, no ?

Reply:
When you have a large codebase and you start looking for something, I can assume you that there is nothing redundant. Because looking for `requestResponse` will get you a lot of responses. And the majority will not be related to the ContentScript. So you'll have to look for `requestInterceptor` and then you'll find a few places where it's used. And then you'll have to read those places to look for `requestResponse`. 

I know that is this the idea of the grep test (https://jamie-wong.com/2013/07/12/grep-test/) taken to the extreme, but this is how I can navigate into the all cozy's codebase.